### ISSUE TITLE
haskintex didn't produce the expected tikz outputs. 

### ISSUE THREAD

**Issue by umeshu** [Daniel-Diaz/haskintex#13]
I tried the sine.htex example under haskintex-0.8.0.0 and the resulted tex file didn't contain the intended tikz codes, as shown below. This was also the case for hatex.htex -- haskintex didn't produced the tikz bits.  Is this a bug in haskintex or the examples are simply outdated?

Output: haskintex sine.htex
```
\documentclass{article}
\author{Mr. Trigonometric}
\title{The sine function}
\usepackage{tikz}
\begin{document}
\maketitle


\end{document}
```
It seemed haskintex completely ignored:
```
\begin{writehaskell}
import Text.LaTeX
import Text.LaTeX.Packages.TikZ.Simple

graphLength :: Double
graphLength = 3*pi

sineGraph :: Figure
sineGraph = Figures [
    Colored (BasicColor Red) $ Line [ (-1,0) , (graphLength,0) ]
  , Colored (BasicColor Red) $ Line [ (0,-1) , (0,1) ]
  , LineWidth (Pt 2) $ pathImage 0.01 (0,graphLength) $ \x -> (x,sin x)
    ]
\end{writehaskell}
\hatex{tikzpicture $ figuretikz sineGraph}

```
 Thanks.

**Comment by Daniel-Diaz** [comment]
Have you tried to run it with the `-verbose`  flag?

My guess would be that the [HaTeX](https://hackage.haskell.org/package/HaTeX) library is not in scope when compiling your code, so `\hatex{tikzpicture $ figuretikz sineGraph}` fails and produces no output.

**Comment by umeshu** [comment]
Thanks @Daniel-Diaz for your response. Yes indeed, when I ran `haskintex -verbose`, it gave the following error report which basically showed  HaTeX library was not in scope,
```
Info: memotree does not exist.
-> Using empty memotree.
Enabled flags: verbose.
Reading sine.htex...
Creating Haskell source file Haskintex_sine.hs...
Evaluating expressions in sine.htex...
Evaluation (LaTeX): tikzpicture $ figuretikz sineGraph
No sandbox or stack project detected.
Warning: Error while evaluating the expression.
Compiler error:
Haskintex_sine.hs:5:1: error:
    Could not find module ‘Text.LaTeX’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
Haskintex_sine.hs:6:1: error:
    Could not find module ‘Text.LaTeX.Packages.TikZ.Simple’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
Haskintex_sine.hs:5:1: error:
    Could not find module ‘Text.LaTeX’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
Haskintex_sine.hs:6:1: error:
    Could not find module ‘Text.LaTeX.Packages.TikZ.Simple’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
Writing final file at sine.tex...
Removing Haskell source file Haskintex_sine.hs (use -keep to avoid this)...
End of processing of file sine.htex
```
However, this is surprising as I have installed `HaTeX`, and compiled its examples without any problem. I installed `haskintex` and `HaTeX` globally using `stack` and was able to call `HaTeX` in `GHCI` without any issue. 

Any suggestion what steps I should try?

**Comment by olafklinke** [comment]
Same problem here. Downloaded haskintex, built+installed with stack. Globally installed HaTeX so that `stack ghci` can import `Text.LaTeX` and `stack path --snapshot-pkg-db` contains a HaTeX directory. The latter is a command evaluated by haskintex in the `loadStackDBPaths` function. Yet either with or without flag `-stackdb` no mention of any sandbox is given when using `-verbose`, and the examples fail with message
```
    Could not find module ‘Text.LaTeX’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
```
See where haskintex looks for packages in `ghci Haskintex.hs`:
```
c = readConf ["-stackdb"]
runH = flip evalStateT c :: Haskintex a -> IO a
runH loadStackDBPaths
```
This turns up the global package dir for the ghc version haskintex was compiled with, which is not necessarily the ghc version the global stack configuration uses. Still, if the ghc versions match, and HaTeX is on one of the paths, the above error is thrown.

**Comment by Daniel-Diaz** [comment]
This seems to have been fixed by #14. If the issue persists, please re-open.
